pkg/gadgets/audit/seccomp/types/types.go

```diff
@@ -15,17 +15,22 @@
 package types
 
 import (
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/columns"
 	eventtypes "github.com/inspektor-gadget/inspektor-gadget/pkg/types"
 )
 
 type Event struct {
 	eventtypes.Event
 
-	Syscall   string `json:"syscall,omitempty"`
-	Code      string `json:"code,omitempty"`
-	Pid       uint32 `json:"pid,omitempty"`
-	MountNsID uint64 `json:"mntns,omitempty"`
-	Comm      string `json:"comm,omitempty"`
+	Pid       uint32 `json:"pid,omitempty" column:"pid,template:pid"`
+	Comm      string `json:"comm,omitempty" column:"comm,template:comm"`
+	Syscall   string `json:"syscall,omitempty" column:"syscall,template:syscall"`
```

Comment:
I think this should use the template introduced in https://github.com/inspektor-gadget/inspektor-gadget/pull/1133/commits/9888e3634f167c69678ce302977b09711fce9e4f and not the one you implemented in the commit before?

Reply:
I used the new template added for traceloop: 

https://github.com/inspektor-gadget/inspektor-gadget/blob/46b7d814d7aadd69cf80c65e6f8f5297b89dc823/pkg/types/types.go#L44-L48